integration-tests/validation/src/main/java/com/android/designcompose/testapp/validation/examples/AllExamples.kt

```diff
@@ -99,6 +101,11 @@ val EXAMPLES: ArrayList<Triple<String, @Composable () -> Unit, String?>> =
         Triple("Text Inval", { TextResizingTest() }, TextResizingTestDoc.javaClass.name),
         Triple("Shared Customization", { ModuleExample() }, ModuleExampleDoc.javaClass.name),
         Triple("Variable Modes", { VariableModesTest() }, VariablesTestDoc.javaClass.name),
+        Triple(
+            "State Customizations",
+            @TargetApi(28) { StateCustomizationsTest(Clock.systemDefaultZone()) },
```

Comment:
What's going on here? Why the TargetApi?

Reply:
systemDefaultZone() is available on api level 28. The app min api level is 30 so I think @TargetApi is good enough here. Also add @TargetApi then Android Studio won't show the warnings/errors.

Reply:
I'm a bit confused why it's showing a warning there, but I dunno, it's probably not causing any issues.